var/spack/repos/builtin/packages/7zip/package.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+import os
+import platform
+import re
+import shutil
+
+from spack import *
+
+
+class _7zip(SourceforgePackage, Package):
+    """7-Zip is a file archiver for Windows"""
+
+    homepage = "https://sourceforge.net/projects/sevenzip"
+    sourceforge_mirror_path = 'sevenzip/files/7z2107-src.tar.xz'
+
+    executables = ['7z']
+
+    version('21.07', sha1='fa288b643575e55531929e7a7759b6884982556d')
```

Comment:
All new package versions need to use a sha256 sum vs. a sha1 (one of the unit tests specifically checks that)

Reply:
I had noticed that failure. I'll re-checksum.